modules/editorial-metadata/lib/components/editorial-metadata-manager.js

```diff
@@ -150,15 +154,17 @@ export default function EditorialMetadataManager( {
 														boxShadow: 'inset 0 0 0 1px #b32d2e',
 													} }
 												></Button>
-												<Button
-													size="compact"
-													variant="primary"
-													icon={ 'edit' }
-													onClick={ () => {
-														setEMetadataTerm( eMetadataTerm );
-														setIsCreateEditModalVisible( true );
-													} }
-												></Button>
+												{
```

Comment:
Is this scope doing anything?

Reply:
Nope, I had been doing some testing with layouts and I must have not reverted this